### PR TITLE
adds talk proposal for Chirayu and Pascal

### PR DESCRIPTION
**Pull request by PascalPrecht**
Please don't merge yet. @Chirayuk needs to review this.

### CODE REVIEW

**Comment by chirayuk** [comment]
Thanks @PascalPrecht.  That looks great.  I added added some suggestions as inline code comments.  Feel free to use none/some/all of them :)

File: better-i18n-for-your-angular-apps/proposal.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+Session:
+  Title: Better i18n for your Angular apps
+  Description: >
+    Internationalization and Localization are the two fundamental procedures
```
Comment (chirayuk) [inline better-i18n-for-your-angular-apps/proposal.yaml]: Internationalization and Localization are fundamental to making your
    applications available in different languages and cultures. This is fairly
    involved and would ideally have a tightly integrated toolchain and workflow
    to support the entire process. Angular currently only offers some
    rudimentary support.  While the community has built some great extensions
    to help in this area, we would really like to have first class support from
    the framework.  In this talk, Chirayu and Pascal are going to show you
    their roadmap for a to bring a great i18n experience to the Angular framework.

File: better-i18n-for-your-angular-apps/proposal.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  Length: 20 # sessions are 5 or 20 minutes long
+  Tags:
+    - i18n
+
+Speaker:
+  name: Chirayu Krishnappa
+  twitter: 
+  email: chirayu@google.com
+  bio: >
+    Software Engineer on the core Angular team at Google.
```
Comment (chirayuk) [inline better-i18n-for-your-angular-apps/proposal.yaml]: Chirayu is a core Angular team member and has contributed to AngularJS,
    AngularDart and Protractor. He loves technology. When not working, he's
    learning music, improv or painting.

**Comment by geddski** [comment]
@PascalPrecht no problem just let me know when ready to merge.

**Comment by PascalPrecht** [comment]
@geddski @chirayuk alrighty, changes applied, squashed and force pushed. Ready to merge

**Pull request merged**